src/ONYXKEYS.ts

```diff
@@ -215,8 +215,8 @@ const ONYXKEYS = {
     /** The NVP containing all information related to educational tooltip in workspace chat */
     NVP_WORKSPACE_TOOLTIP: 'workspaceTooltip',
 
-    /** Whether to hide save search rename tooltip */
-    NVP_SHOULD_HIDE_SAVED_SEARCH_RENAME_TOOLTIP: 'nvp_should_hide_saved_search_rename_tooltip',
+    /** Whether to show save search rename tooltip */
+    NVP_SHOULD_SHOW_SAVED_SEARCH_RENAME_TOOLTIP: 'nvp_should_show_saved_search_rename_tooltip',
```

Comment:
I think we should drop the NVP part since we wouldn't save this to the DB and it's just a local value
```suggestion
    SHOULD_SHOW_SAVED_SEARCH_RENAME_TOOLTIP: 'shouldShowSavedSearchRenameTooltip',
```